### ISSUE TITLE
Is this project still alive? 

### ISSUE THREAD

**Issue by joshuaebowling** [emercoin/emercoin#101]
I see there hasn't been a release in over a year. I'd like to hodl and make use of these services but I'd like to know there's still active development/interest beyond my own.
Thanks!

**Comment by maxihatop** [comment]
Yes, project alive, and last commit to codepase had been sent yesterday:

https://github.com/emercoin/emercoin/commits/master

**Comment by NESS-Network** [comment]
> I see there hasn't been a release in over a year. I'd like to hodl and make use of these services but I'd like to know there's still active development/interest beyond my own.
> Thanks!

Very much alive sir, and you're not alone building on it. :)

**Comment by joshuaebowling** [comment]
thank you for the update! I look forward to trying out this remarkable work you all have done!

**Comment by PanderMusubi** [comment]
A new release will showcase that Emercoin is actively maintained and works with the latest libraries and operating systems. This will also help its statistics and ratings, which in turn can be reflected in its value.

**Comment by Jeff-Bouchard** [comment]
I know for a fact that it's actively maintained and code is being updated at this very moment.

**Comment by PanderMusubi** [comment]
This issue can be closed.
